core/eolearn/core/eodata.py

```diff
@@ -185,11 +187,23 @@ def _parse_feature_value(self, value: object, feature_name: str) -> gpd.GeoDataF
             value = gpd.GeoDataFrame(geometry=value, crs=value.crs)
 
         if isinstance(value, gpd.GeoDataFrame):
-            if self.feature_type is FeatureType.VECTOR and FeatureType.TIMESTAMP.value.upper() not in value:
-                raise ValueError(
-                    f"{self.feature_type} feature has to contain a column 'TIMESTAMP' with timestamps but "
-                    f"feature {feature_name} doesn't not have it."
-                )
+            if self.feature_type is FeatureType.VECTOR:
+                if "TIMESTAMP" in value:
+                    warn(
+                        (
+                            f"{self.feature_type} feature has to contain a column 'TIMESTAMPS' instead of 'TIMESTAMP'"
+                            " (deprecated)"
```

Comment:
```suggestion
                            f"{self.feature_type} feature has to contain a column 'TIMESTAMPS' instead of 'TIMESTAMP'."
                            " In the future the old name will no longer be supported."
```